js/app.js

```diff
@@ -4,32 +4,52 @@
     for(var key in project){
       this[key] = project[key];
     }
+    this.projectTotal = Project.projectLines(this.code);
   }
 
   Project.all = [];
 
+  //Method that compiles the project template and returns it.
   Project.prototype.toHtml = function(){
     var $source = $('#project-template').html();
     var template = Handlebars.compile($source);
     return template(this);
   };
 
+  //Method that populates the Project.all property
   Project.loadAll = function(data){
-    data.forEach(function(ele) {
+    data.map(function(ele) {
       Project.all.push(new Project(ele));
     });
   };
 
+  //Method that is used to populate the projectTotal property
+  Project.projectLines = function(code){
+    return code.reduce(function(a, b){
+      return a + b;
+    });
+  };
+
+  //Method that is used to calculate the total lines of code across projects, it is called by the stats template.
   Project.totalLines = function(){
     return Project.all.map(function(project){
-      return project.code.reduce(function(a, b){
-        return a + b;
-      });
+      return project.projectTotal;
     }).reduce(function(a, b){
       return a + b;
     }, 0);
   };
 
+  //Method to grab each project's total lines into a format that is usable for a handlebars template
+  Project.collateTotals = function(){
+    return Project.all.map(function(project){
+      // console.log(typeof(project.projectTotal));
+      return ('<li>' + project.projectName + ': '+ project.projectTotal + ' lines</li>');
```

Comment:
If you're planning on using Handlebars, pulling this into an object you can later pass in to Handlebars is your best bet.